CMakeLists.txt

```diff
@@ -52,15 +52,16 @@ etn_target(shared ${PROJECT_NAME}
         fty_common_dto
         fty_common_logging
     TARGET_DESTINATION
-        ${CMAKE_INSTALL_PREFIX}/lib/bios
+        ${CMAKE_INSTALL_LIBDIR}
```

Comment:
Are you sure ? by default CMAKE_INSTALL_LIBDIR=lib

Reply:
yes LIBDIR debian == /usr/lib/x86....